src/genie/libs/parser/junos/show_security.py

```diff
@@ -0,0 +1,112 @@
+"""show_security.py
+JunOS parsers for the following show commands:
+    * show security policies hit-count
+"""
+# Python
+import re
+
+# Metaparser
+from genie.metaparser import MetaParser
+from pyats.utils.exceptions import SchemaError
+from genie.metaparser.util.schemaengine import (Any, Optional, Use, Schema, ListOf)
+
+
+class ShowSecurityPoliciesHitCountSchema(MetaParser):
+    """ Schema for:
+            * show security policies hit-count
+    """
+    """schema = {
+        "security_policy_counts": {
+            str: {
+                "security_policy": ListOf(
+                    {
+                        "index": str,
+                        "from_zone": str,
+                        "to_zone": str,
+                        "name": str,
+                        "policy_hit_count": str
+                    },
+                )
+                "total_policies": str
+            }
+        }
+    }"""
+
+    # Main Schema
+    schema = {
+        "security_policy_counts": {
+            str: {
```

Comment:
Minor, but because str is a key here it's better practice to use Any(): but not required.